03/app.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import MenuItems from "./components/MenuItems";
+
+ReactDOM.render(
+    <MenuItems text="kontakt" url ="/contact/"/>,
```

Comment:
Raczej zrobiłbym to jako `<MenuItem>`, który zawiera tylko `<li>` bo 1 element menu.
Dopiero ten `<MenuItem>` wstawiłbym do `<ul>` czyli `<ul><MenuItem></ul>`